riscv/mmu.h

```diff
@@ -184,27 +184,32 @@ class mmu_t
       } \
   }
 
+  #define amo_or_zicbom_op(BODY) \
+    try { \
+      BODY \
+    } catch (trap_load_address_misaligned& t) { \
+      /* AMO faults should be reported as store faults */ \
```

Comment:
AMO/Zicbom here, not just AMO

Reply:
misaligned fault seems not needed for Zicbom.